naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/GraphQLController.scala

```diff
@@ -0,0 +1,92 @@
+package org.coursera.naptime.ari.graphql
+
+import javax.inject._
+
+import com.typesafe.scalalogging.StrictLogging
+import org.coursera.naptime.ari.EngineApi
+import org.coursera.naptime.ari.Response
+import play.api.libs.json.JsObject
+import play.api.libs.json.JsString
+import play.api.libs.json.Json
+import play.api.mvc._
+import sangria.execution.ErrorWithResolver
+import sangria.execution.Executor
+import sangria.execution.QueryAnalysisError
+import sangria.parser.QueryParser
+import sangria.renderer.SchemaRenderer
+import sangria.marshalling.playJson._
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+
+@Singleton
+class GraphQLController @Inject() (
+  graphqlSchemaProvider: GraphqlSchemaProvider,
```

Comment:
if we're following our coding style, then this should be 4 space indented, and `withStrictLogging` should be two-space indented